hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/json/GroupConditionsInfo.java

```diff
@@ -35,7 +39,11 @@
  * @author lucas ponce
  */
 public class GroupConditionsInfo {
+
+    @JsonInclude(Include.NON_EMPTY)
```

Comment:
I put it to avoid conditions = null values during json deserialization.
It is not a big deal we can leave it as it or just @JsonInclude.